panoptes_client/workflow.py

```diff
@@ -115,6 +117,22 @@ def remove_subject_sets(self, subject_sets):
 
         return self.links.subject_sets.remove(subject_sets)
 
+    def subject_set_status(self, subject_set_id):
```

Comment:
i'd keep the function name consistent with the returned resource and the function on SubjectSet resource too. Trying to add the `subject_set` behaviour  / intention here is a mouthful too - perhaps the param captures this enough? 
`def subject_set_subject_workflow_statuses(self, subject_set_id)`
Feel free to push back on this.
```suggestion
    def subject_workflow_statuses(self, subject_set_id):
```

Reply:
I like your suggestion, its clear and concise. I'll update